presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -10,6 +10,30 @@ Linux Operating System
 
 * 64-bit required
 * newer release preferred, especially when running on containers
+* adequate ulimits for the user that runs the Presto process. These limits 
+  may depend on the specific Linux distribution you are using. Currently,
+  Presto requires a minimum of 4096 open file descriptors, and strongly
+  recommends at least 8192 when it runs. However, the number of open file
+  descriptors needed for a particular Presto instance scales with the number
+  of machines in the cluster as roughly ``number of machines^2 * workload``.
+  Many modern Linux systems built to host large applications such as Presto
```

Comment:
This seems needlessly verbose, and is confusing. We mention 32k here, then show completely different numbers below. Also, it's not clear why we would set a soft limit different than the hard limit. I think it's better to simply tell people exactly what to put.

> We recommend using a much higher limit, which can typically be set in `/etc/security/limits.conf`:

Reply:
The soft and hard limits were suggested by Kamil in the slack thread.

Reply:
I understands, but having a hard limit different from the soft limit doesn't make sense for our use case. The soft limit is enforced as the limit, but it can be increased up to the hard limit.

Reply:
Hi @electrum - Understood. Gathering comments, it looks like the preference is to leave nproc out of the example altogether, and only have a single line that we suggest to add to the limits.conf file. I gather from the above that the line should be the nofile hard limit. Is that correct? What I am not clear on is if there is a consensus among the engineering leads as to what that limit should be. Could you provide what you believe should be the example ulimits entry(ies) with the agreed upon values? Thank you!